README.md

```diff
@@ -1,2 +1,40 @@
 # ansible-python-security
 Ansible Role to install Python security modules.
+
+## Role Variables
+
+- `urllib3_version`: Version of `urllib3` to install. (Default: `"1.21.*"`)
```

Comment:
We may want to namespace all of these variables with a prefix specific to this role, otherwise it could conflict with similarly name variables that have nothing to do with this role.